Makefile

```diff
@@ -52,6 +53,7 @@ cross-local:
 	@echo "🐳 $@"
 	go build -o "bin/dnet-$$GOOS-$$GOARCH" ./cmd/dnet
 	go build -o "bin/docker-proxy-$$GOOS-$$GOARCH" ./cmd/proxy
+	go build -o "bin/dignosticClient-$$GOOS-$$GOARCH" ./diagnose/client
```

Comment:
nit: why put it in /diagnose/ and not /cmd/ ? like the others ?

Reply:
yep moved